File: Frontend/src/app/components/configure/models/OIDCConfiguration.ts
export interface OIDCConfiguration {
    id: string | null,
    authority: string | null,
    audience: string | null,
    callback_uri: string | null,
    certificatePassword: string | null,
    certificateSerial: string | null,
    client_id: string | null,
    redirect_uri: string | null,
    response_type: string | null,
    scope: string | null
}

File: OpenIdConnectIntegrationExample/Models/OIDCConfiguration.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace OpenIdConnectIntegrationExample.Models
{
    public class OIDCConfiguration
    {
        [Key]
        public Guid? Id { get; set; }
        public string? authority { get; set; }
        public string? audience { get; set; }
        public string? callback_uri { get; set; }
        public string? certificatePassword { get; set; }
        public string? certificateSerial { get; set; }
        public string? client_id { get; set; }
        public string? client_secret { get; set; }
        public string? jsonWebKey { get; set; }
        public string? redirect_uri { get; set; }
        public string? response_type { get; set; }
        public string? scope { get; set; }
    }
}